src/lucky_migrator/change_null_statement.cr

```diff
@@ -0,0 +1,31 @@
+# Builds an SQL statement for changing a columns NOT NULL status.
+#
+# ### Usage
+#
+# ```
+# ChangeNullStatement.new(:users, :email, make: :optional).build
+# # => "ALTER TABLE users ALTER COLUMN email DROP NOT NULL;"
+# ```
+class LuckyMigrator::ChangeNullStatement
+  ALLOWED_MAKE_OPTIONS = [:optional, :required]
```

Comment:
Instead of allowing symbols and having to catch invalid options, maybe the option could be a Bool called `required` so you’d do: required: true/false

Reply:
Sure, that's even simpler